src/infinistore.cpp

```diff
@@ -655,6 +699,21 @@ void on_read(uv_stream_t *stream, ssize_t nread, const uv_buf_t *buf) {
                             print_ipc_handle(client->local_meta.ipc_handle);
                             handle_request(client);
                             break;
+                        case OP_CHECK_EXIST:
+                            client->key_to_check.assign(client->recv_buffer,
+                                                        client->expected_bytes);
+                            handle_request(client);
+                            break;
+                        case OP_GET_MATCH_LAST_IDX:
+                            INFO("OP_GET_MATCH_LAST_IDX");
+                            if (!deserialize(client->recv_buffer, client->expected_bytes,
+                                             client->keys_meta)) {
+                                printf("failed to deserialize keys meta\n");
```

Comment:
no printf...

Reply:
this is just copy/paste from previous code:) I will fix all printf

Reply:
fixed